aptos-move/move-examples/token_objects/ambassador/client/ambassador.ts

```diff
@@ -0,0 +1,171 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { AptosAccount, AptosClient, HexString } from "aptos";
+
+const NODE_URL = process.env.APTOS_NODE_URL || "https://fullnode.testnet.aptoslabs.com";
+const FAUCET_URL = process.env.APTOS_FAUCET_URL || "https://faucet.testnet.aptoslabs.com";
+
+function fetchGraphQL(
+    operationsDoc: string,
+    operationName: string,
+    variables: Record<string, any>
+) {
+    return fetch('https://indexer-testnet.staging.gcp.aptosdev.com/v1/graphql', {
+        method: 'POST',
+        body: JSON.stringify({
+            query: operationsDoc,
+            variables,
+            operationName,
+        }),
+    }).then(result => result.json());
+}
+
+async function getTokenAddr(ownerAddr: HexString, tokenName: string): Promise<HexString> {
+    const operation = `
+  query TokenOwnership {
+    current_token_ownerships_v2(
+      where: {owner_address: {_eq: "${ownerAddr}"}, amount: {_gt: 0}}
+      order_by: {last_transaction_version: desc}
+    ) {
+      amount
+      current_token_data {
+        token_uri
+        token_data_id
+        token_properties
+        token_name
+        current_collection {
+          collection_name
+        }
+      }
+      token_standard
+    }
+  }
+`;
+    const data = await fetchGraphQL(operation, "TokenOwnership", {});
+    const tokenOwnership = data.data;
+
+    // TODO: `tokenOwnership` can be obtained by the following code when a new SDK version is ready.
+    // const tokenOwnership = await provider.getTokenOwnershipV2(ownerAddr);
```

Comment:
Can you integrate https://github.com/aptos-labs/aptos-core/pull/8210 (might need to change tsconfig.json to depend on local code)? If we can test this out, we can avoid hardcoding the query here. cc @0xmaayan

Reply:
we talked about it. I created an example PR on how to use a local SDK build https://github.com/aptos-labs/aptos-core/pull/8133. 
Once https://github.com/aptos-labs/aptos-core/pull/8210 is merged, we can publish a new version and @junkil-park can use the updated version instead of building from local.

Reply:
Updated to use the published `aptos` sdk.